CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ add_subdirectory(events EXCLUDE_FROM_ALL)
 add_subdirectory(connectivity EXCLUDE_FROM_ALL)
 add_subdirectory(storage EXCLUDE_FROM_ALL)
 add_subdirectory(drivers/device_key EXCLUDE_FROM_ALL)
-add_subdirectory(drivers/source/usb EXCLUDE_FROM_ALL)
+add_subdirectory(drivers/usb EXCLUDE_FROM_ALL)
```

Comment:
Not related to this PR: for consistency with out libraries (e.g. connectivity, storage), `add_subdirectory` of `device_key` and `usb` should be done in `drivers/CMakeLists.txt`.